### DISCUSSION TITLE
PATCH settings with key value

### DISCUSSION THREAD

**Discussion by tvld** [directus/directus#2772] [Feature Requests]
For convenience, I think it would be nice to PATCH settings with a ``key``, ``value`` pair, and not just with the ``id`` number:

```js
PATCH {{host}}/settings HTTP/1.1
content-type: application/json

{
  "key": "project_name",
  "value": "My new project name"
}
```

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
